packages/wrangler/src/api/dev.ts

```diff
@@ -96,6 +98,8 @@ export async function unstable_dev(
 	const {
 		// there are two types of "experimental" options:
 		// 1. options to unstable_dev that we're still testing or are unsure of
+		processEntrypoint,
```

Comment:
```suggestion
		processEntrypoint = false,
```

Would doing this avoid having to use `!!` later on?